http-api/src/commit.rs

```diff
@@ -16,16 +15,25 @@ pub struct CommitsQueryString {
 
 #[derive(Serialize)]
 pub struct CommitTeaser {
-    pub header: Header,
+    pub header: git::Commit,
     pub context: CommitContext,
 }
 
+/// Commit statistics.
+#[derive(Clone, Serialize)]
+pub struct CommitStats {
```

Comment:
Wouldn't a commit only have stats compared to another commit? Or do you always calculate this based on the parent? :thinking:

Reply:
Yeah we only refer to these stats compared to another commit, with [7b2c38a](https://github.com/radicle-dev/radicle-client-services/pull/196/commits/7b2c38a40245b6c121068547b1d6a1b22b84ee1c) it's clearer since we only use it in the context of a `radicle_surf::diff::Diff`